errors/inline-script-id.md

```diff
@@ -0,0 +1,26 @@
+# next/script components with inline content require an `id` attribute
+
+## Why This Error Occurred
+
+`next/script` components with inline content require an `id` attribute for deduplication.
```

Comment:
Please replace with 
`next/script` components with inline content require an `id` attribute to be defined to track and optimize the script.

to be in sync [with the docs](https://github.com/vercel/next.js/blob/canary/docs/basic-features/script.md)

Reply:
```suggestion
`next/script` components with inline content require an `id` attribute to be defined to track and optimize the script.
```